test/sql/hnsw_select.sql

```diff
@@ -36,11 +46,28 @@ WITH neighbors AS (
 RESET client_min_messages;
 
 -- Verify where condition works properly and still uses index
-SELECT * FROM small_world WHERE b IS TRUE ORDER BY v <-> '{0,0,0}';
-EXPLAIN (COSTS FALSE) SELECT * FROM small_world WHERE b IS TRUE ORDER BY v <-> '{0,0,0}';
+SELECT has_index_scan('SELECT * FROM small_world WHERE b IS TRUE ORDER BY v <-> ''{0,0,0}''');
```

Comment:
Are multiple single quotes in `''{0,0,0}''` intentional?

Reply:
Yes. Otherwise, it would match with the single quote that starts the string.